src/newLaunch/seed/stages/stage4.ts

```diff
@@ -152,6 +152,8 @@ export class Stage4 extends BaseStage<ISeedConfig> {
       message = "Please enter a number greater than zero for the Funding Target";
     } else if (!this.launchConfig.launchDetails.fundingMax || this.launchConfig.launchDetails.fundingMax === "0") {
       message = "Please enter a number greater than zero for the Funding Maximum";
+    } else if (this.launchConfig.tokenDetails.projectTokenAddress === this.launchConfig.launchDetails.fundingTokenAddress) {
+      message = "Funding Token and Project Token cannot be same. Please change one of the addresses.";
```

Comment:
Better text because funding tokens are not selected by address:

"Funding Token and Project Token cannot be same. Please reenter one or the other."

Reply:
done